### DISCUSSION TITLE
Dynamically change datasource based on the selected dropdown option

### DISCUSSION THREAD

**Discussion by dr3nyl** [Power-Components/livewire-powergrid#921] [Q&A]
Hi, I'm thinking if there is a way to dynamically change the data source depending on the selected option in a select tag?

Example: passing the selected dropdown value to the data source to change the "Where" clause and update the data source

**Comment by kolydart** [comment]
I would argue that the opposite should be implemented. Make current datasource available to the Filter() facade.

That would be a great feature, if it exists! It would allow the creation of "faceted" type of search. Current criteria affect available (select) filters, so that they contain only values applicable to the curren sub-set.

In my opinion this would sky-rocket the usability of the package.
